### DISCUSSION TITLE
Convert vector vb to text

### DISCUSSION THREAD

**Discussion by neeraj-mahapatra** [milvus-io/milvus#25315] [Q&A and General discussion]
How we can convert vector db to text format?

**Comment by yhmo** [comment]
"convert vector db to text format"------------------ I don't think this is possible.
Do you mean "convert vector to text"? Once a string is converted to a vector, the vector has lost the original meaning of the string, and not possible to be converted to the original string.

**Reply by neeraj-mahapatra** [reply]
Ok! Hey, thanks for answering
